extensions/terraform/extension.toml

```
@@ -1,7 +1,7 @@
 id = "terraform"
 name = "Terraform"
 description = "Terraform support."
-version = "0.0.4"
+version = "0.0.5"
```

We bump the extension versions separately from changes:

```suggestion
version = "0.0.4"
```